Rocket.Chat/ChangeAppIconViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  ChangeAppIconViewController.swift
+//  Rocket.Chat
+//
+//  Created by Artur Rymarz on 08.02.2018.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+final class ChangeAppIconViewController: UIViewController {
+
+    private let viewModel = ChangeAppIconViewModel()
+
+    @IBOutlet private weak var labelHeaderTitle: UILabel! {
+        didSet {
+            labelHeaderTitle.text = viewModel.header
+        }
+    }
+
+    @IBOutlet private weak var collectionView: UICollectionView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        title = viewModel.title
+        collectionView.delegate = self
+        collectionView.dataSource = self
+    }
+
+    private func changeIcon(name: String) {
+        let iconName = name == "Default" ? nil : name
+
+        if #available(iOS 10.3, *) {
+            UIApplication.shared.setAlternateIconName(iconName) { error in
+
+                guard let error = error else {
+                    return
+                }
+
+                self.reportError(message: (error as NSError).localizedDescription)
+            }
+        } else {
+            reportError(message: viewModel.iosVersionMessage)
+        }
+    }
+
+    private func reportError(message: String?) {
+        let alert = UIAlertController(title: viewModel.errorTitle,
+                                      message: message,
+                                      preferredStyle: .alert)
+
+        let done = UIAlertAction(title: "OK", style: .default, handler: nil)
+        alert.addAction(done)
+
+        present(alert, animated: true)
```

Comment:
I just refactored all our files to use the Helper/Alert.swift. Please do so too, to avoid redundancy.